### PR TITLE
clblast: update to use modern cmake syntax

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/c/clblast.rb [hunk 1]
```diff
@@ -24,8 +24,10 @@ class Clblast < Formula
   end
 
   def install
-    system "cmake", ".", *std_cmake_args, "-DCMAKE_INSTALL_RPATH=#{rpath}"
```
Comment (ZhongRuoyu) [inline Formula/c/clblast.rb:27]: `CMAKE_INSTALL_RPATH` should be kept?

Comment (chenrui333) [reply Formula/c/clblast.rb:27]: it did not fail the audit check?

Comment (ZhongRuoyu) [reply Formula/c/clblast.rb:27]:
```
Files with missing rpath:
  /usr/local/Cellar/clblast/1.6.2/bin/clblast_tuner_routine_xgemm
  /usr/local/Cellar/clblast/1.6.2/bin/clblast_tuner_routine_xtrsv
```

Comment (chenrui333) [reply Formula/c/clblast.rb:27]: sorry, I missed it. adding it back now.

**Review by ZhongRuoyu — APPROVED** [review]

**Pull request merged**